fem/integ/bilininteg_mass_kernels.cpp

```diff
@@ -25,16 +25,13 @@ static void PAMassAssembleDiagonal1D(const int NE,
                                      const int D1D,
                                      const int Q1D)
 {
-   MFEM_VERIFY(D1D <= MAX_D1D, "");
-   MFEM_VERIFY(Q1D <= MAX_Q1D, "");
    auto B = Reshape(b.Read(), Q1D, D1D);
    auto D = Reshape(d.Read(), Q1D, NE);
    auto Y = Reshape(y.ReadWrite(), D1D, NE);
    mfem::forall(NE, [=] MFEM_HOST_DEVICE (int e)
    {
       for (int dx = 0; dx < D1D; ++dx)
       {
-         Y(dx, e) = 0.0;
```

Comment:
This is bugfix, right? This method should perform `+=`, not `=`.

Reply:
Yes, this was a bug fix from 04d34e4de159423bff299e53a57772d725e05011